CondFormats/TotemReadoutObjects/interface/TotemDAQMapping.h

```diff
@@ -0,0 +1,52 @@
+/****************************************************************************
+*
+* This is a part of TOTEM offline software.
+* Authors: 
+*   Maciej Wróbel (wroblisko@gmail.com)
+*   Jan Kašpar (jan.kaspar@cern.ch)
+*
+****************************************************************************/
+
+#ifndef _TotemDAQMapping_h_
+#define _TotemDAQMapping_h_
+
+#include "DataFormats/TotemRawData/interface/TotemFramePosition.h"
```

Comment:
TotemFramePosition looks like the only "DataFromats/" object included here.
Looking at it, the class describes the hardware channel ID, something that is fixed for that geometry or logical hardware element and doesn't change between events.
In other detectors we have such objects in CondFormats/[DetectorName]Objects